README.md

```diff
@@ -1989,6 +1989,11 @@ mdls /path/to/file
 
 ### System Integrity Protection
 
+#### Check SIP Status
```

Comment:
This should be folded into the other examples to enable and disable it. It's an oversight that this is not the case already. I will take care of it.

Reply:
Done.

Reply:
I have already added this since I was reworking the list today. In resolving the marge conflicts please use the master version in bulk.